packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -66,6 +66,7 @@ import {
 import { IMarkdownEmitterContext } from '@microsoft/api-documenter/lib/markdown/MarkdownEmitter';
 import { AstDeclaration } from '@microsoft/api-extractor/lib/analyzer/AstDeclaration';
 import { paths as cliPaths } from '../../lib/paths';
+import { resolvePackagePath as resolvePackagePathBackend } from '@backstage/backend-common';
```

Comment:
Let's not add a dependency on this package. I don't think we need to either, you should be able to use `require.resolve` directly.